lib/LaTeXML/Package/eTeX.pool.ltxml

```diff
@@ -45,6 +45,11 @@ DefMacro('\readline Number SkipKeyword:to SkipSpaces Token', sub {
       DefMacroI($token, undef, Tokens(Explode(($mouth->readRawLine || '') . "\r"))); }
     return; });
 
+# https://tex.loria.fr/moteurs/etex_ref.html#scantokens
+# TODO: "Parentheses and a single space representing the pseudo-file will be displayed
+#        if \tracingscantokens (q.v.) is positive and non-zero."
+#       We are currently missing a final trailing space in some uses
+#           c.f. t/tokenize/hashes.tex
```

Comment:
Of course this is more complicated for additional reasons, but won't address that in this PR if possible. Left a guiding comment.